src/const/urls.ts

```diff
@@ -3,7 +3,7 @@ import type { SitemapPage } from '@/types/sitemap';
 export const JUMPER_URL = 'https://jumper.exchange';
 export const LIFI_URL = 'https://li.fi';
 export const EXPLORER_URL = 'https://explorer.li.fi';
-export const DISCORD_URL = 'https://discord.gg/lifi';
+export const DISCORD_URL = 'https://discord.gg/jumperexchange';
```

Comment:
Can we please use this const everywhere we have this link, as per Adrian's comment on the other PR?

Reply:
@chybisov `DISCORD_URL` is being already used, although there is another link with invite -- i just added the other one as `const` and replaced the link with that `const`